data-tool/flows/tombstone/tombstone_queries.py

```diff
@@ -270,7 +270,7 @@ def get_business_query(corp_num, suffix):
     --
         c.send_ar_ind,
         c.last_ar_reminder_year,
-        to_char(c.last_ar_filed_dt::timestamptz at time zone 'UTC', 'YYYY-MM-DD HH24:MI:SSTZH:TZM') as last_ar_date,
+        to_char(c.last_ar_filed_dt, 'YYYY-MM-DD') as last_ar_date,
```

Comment:
ar date is saved in PST https://github.com/bcgov/lear/blob/main/queue_services/entity-filer/src/entity_filer/filing_processors/annual_report.py#L34